src/rust/e2e-tests/tests/sysmon_log_e2e_test.rs

```diff
@@ -140,6 +141,23 @@ async fn test_sysmon_log_e2e(ctx: &mut E2eTestContext) -> eyre::Result<()> {
     ))
     .await;
 
+    let execution_hits_scanner_handle = KafkaTopicScanner::new(
+        ConsumerConfig::with_topic("analyzer-executions"),
```

Comment:
(fwiw, this topic is probably poorly named but matches the one defined in grapl-infrastructure
```
            # This list must match the set of all topics specified in
            # grapl-infrastructure/pulumi/confluent_cloud/index.ts
```
)